rspack/lib/index.d.ts

```diff
@@ -0,0 +1,16 @@
+import * as RspackCore from '@rspack/core'
+import { NapiNextExternalsPluginOptions } from '@next/rspack-binding'
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+declare class NextExternalsPlugin {
+  /**
+   * The banner text to be added to the output file.
```

Comment:
nitpick: might need to replace this boilerplate comment with the actual usage here.